damus/Views/FollowingView.swift

```diff
@@ -52,6 +54,7 @@ struct FollowersView: View {
                 }
             }
         }
+        .padding(.horizontal)
```

Comment:
I believe the padding modifier should be attached to the LazyVStack to avoid moving the scroll indicator. In my opinion, it should also be horizontal and vertical. Add all changes to the FollowingView as well.

![IMG_4465](https://user-images.githubusercontent.com/19398259/209706172-136afa91-035f-43c0-bd82-2d1c738eaf79.PNG)

Reply:
Thanks. Fixed.